src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -1117,15 +1117,15 @@
   - name: Upload an entire directory to Azure Blob using url.
     text: az storage copy -s /path/to/dir -d https://[account].blob.core.windows.net/[container]/[path/to/directory] --recursive
   - name: Upload an entire directory to Azure Blob using account name and container name.
-    text: az storage copy --source-local-path /path/to/dir --destination-account-name mystorageaccount --destination-container mycontainer --recursive
+    text: az storage copy -s /path/to/dir --destination-account-name mystorageaccount --destination-container mycontainer --recursive
```

Comment:
There are two places that haven't been changed. Are they in line with expectations?
https://github.com/Azure/azure-cli/blob/cfa31a7439e3d7dd4ff38fe03afd8a753a9cb19b/src/azure-cli/azure/cli/command_modules/storage/_help.py#L1158
https://github.com/Azure/azure-cli/blob/cfa31a7439e3d7dd4ff38fe03afd8a753a9cb19b/src/azure-cli/azure/cli/command_modules/storage/_help.py#L1114

Reply:
done